src/jobsautoreport/query.py

```diff
@@ -73,33 +76,38 @@ def _get_query_unsuccessfull_jobs(from_date: datetime, to_date: datetime) -> dic
     def query_successfull_jobs(
         self, from_date: datetime, to_date: datetime
     ) -> list[JobDetails]:
-        res_successfull_jobs = self._os_client.search(
-            body=self._get_query_successfull_jobs(from_date, to_date), index=self._index
-        )
+        query = self._get_query_successfull_jobs(from_date, to_date)
+        logger.debug("OpenSearch query:\n%s", query)
+        res_successfull_jobs = self._os_client.search(body=query, index=self._index)
+        logger.debug("OpenSearch query:\n%s", res_successfull_jobs)
```

Comment:
to remove some of the clutter and repetition maybe this can be extracted into a method